docs/fml-spec.md

```diff
@@ -0,0 +1,576 @@
+# The Feature Manifest Language
```

Comment:
Should we have a section that talks about Desktop and plans with that - maybe a link to https://experimenter.info/desktop-feature-api/#registering-a-new-feature?